store/snapshots/manager.go

```diff
@@ -484,4 +488,56 @@ func (m *Manager) snapshot(height int64) {
 
 		m.logger.Debug("pruned state snapshots", "pruned", pruned)
 	}
+
+	if os.Getenv("SNAPSHOT_EXPORT_ENABLED") == "1" {
+		m.logger.Info("exporting state snapshot", "height", height)
+		if err := m.ExportSnapshot(snapshot); err != nil {
+			m.logger.Error("failed to export snapshot", "height", height, "err", err)
+		}
+	}
+}
+
+// ExportSnapshot exports snapshot chunk to specified directory
+func (m *Manager) ExportSnapshot(snapshot *snapshottypes.Snapshot) error {
```

Comment:
chunks are already stored in a directory, unclear why another location is needed? Can streamingfast use the exiting location?